src/borg/helpers/fs.py

```diff
@@ -245,7 +245,10 @@ def remove_dotdot_prefixes(path):
 
     `path` is expected to be normalized already (e.g. via `os.path.normpath()`).
     """
-    path = path.lstrip("/")
+    if is_win32:
+        path = path.replace(":", "").replace("\\", "/")
```

Comment:
guess this would also affect colons used for alternate data streams (not sure how borg does or should deal with these).

so better check for a colon at index 1 and only then replace **one** of them.